templates/types/streaming/fastapi/README-template.md

```diff
@@ -92,6 +92,24 @@ docker run \
   <your_backend_image_name>
 ```
 
+## Deployments
+
+### Deploy to [Fly.io](https://fly.io/):
+
+First, check out the [flyctl installation guide](https://fly.io/docs/flyctl/install/) and install it to your machine then authenticate with your Fly.io account:
+
+```shell
+fly login
+```
+
+Then, run this command and follow the prompts to deploy the app.:
+
+```shell
+fly launch --internal-port 8000
+```
+
+- Note: Make sure all the needed environment variables in the [.env](.env) file (e.g. `OPENAI_API_KEY`) are set.
```

Comment:
So it's using the .env file? If so, how about adding this note before the deployment?